src/System.Security.Cryptography.DeriveBytes/src/System/Security/Cryptography/DeriveBytes.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Security.Cryptography
+{
+    public abstract class DeriveBytes : IDisposable
+    {
+        protected DeriveBytes()
```

Comment:
Is this ctor necessary?  Without it, it'll get a default ctor, and since the type is abstract, that's effectively the same as protected, right?